.github/workflows/build-crates-individually.patch.yml

```diff
@@ -0,0 +1,76 @@
+name: Build crates individually
+
+# We need to keep the `matrix` job in this workflow as-is, as we need the results
+# to actually match the same `build` job names from the original file.
+on:
+  pull_request:
+    paths-ignore:
+      # production code and test code
+      - '**/*.rs'
+      # dependencies
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+      # workflow definitions
+      - '.github/workflows/build-crates-individually.yml'
+
+env:
+  CARGO_INCREMENTAL: 0
+  RUST_BACKTRACE: full
+  RUST_LIB_BACKTRACE: full
+  COLORBT_SHOW_HIDDEN: '1'
+
```

Comment:
I don't think these env vars are needed, because we don't actually build anything
```suggestion
```